src/caret_analyze/plot/bokeh/callback_info.py

```diff
@@ -56,6 +58,10 @@ def _create_latency_df(
         callback: CallbackBase
     ) -> pd.DataFrame:
         df = callback.to_dataframe()
+        if len(df) == 0:
+            logger.warning('Since no timestamp is recorded, '
+                           'the latency cannot be calculated. '
+                           f'callback_name: {callback.callback_name}')
```

Comment:
This implementation is redundant.
It's preferred to move error cases to "_to_dataframe_core".

```
for cb in self._callbacks:
    try:
        latency_df = self._create_latency_df(xaxis_type, cb)
        concat_latency_df = pd.concat([concat_latency_df, latency_df],
                                      axis=1)
    exception IndexError as e:
        df = cb.to_dataframe() 
        if (len(df) == 0) :
            logger.warning()
```

Reply:
@hsgwa 
I have fixed by https://github.com/tier4/CARET_analyze/pull/130/commits/3063b537f2c6a302c8da0fbcdd4d3e2a41558bea and https://github.com/tier4/CARET_analyze/pull/130/commits/a9a63f5caa4828692d6cb8b0ae59f51070d850cf.
Could you please review it again?

Reply:
Sorry, that wasn't good advice.
To guarantee  existence of source_df column, following implementation seems better.

```
for cb in self._callbacks:
    try:
        latency_df = self._create_latency_df(xaxis_type, cb)
        concat_latency_df = pd.concat([concat_latency_df, latency_df],
                                      axis=1)
    except IndexError as e:
        pass # or logger.warning
   finally:
        if column in source_df:
            pass # insert empty df
        if (len(cb.to_dataframe() ) == 0) :
            logger.warning()
```

Reply:
@hsgwa 
Fixed in https://github.com/tier4/CARET_analyze/pull/130/commits/a04cae08875770c39a2f5fd7642b943b07b55740 :bow: